package.json

```diff
@@ -8,8 +8,8 @@
   },
   "scripts": {
     "lint": "node ./node_modules/semistandard/bin/cmd.js",
-    "prepublish": "node downloadCurrentVersion.js && node verifyVersion.js",
-    "pretest": "npm run lint",
+    "prepublishOnly": "node downloadCurrentVersion.js && node verifyVersion.js",
+    "pretest": "npm run lint && npm run prepublishOnly",
```

Comment:
This now tries to run ``verifyVersion.js`` before our ``t_ems_solcjs`` tests triggered from the solidity repo with CI build soljson.js - which fails the version test...